src/NServiceBus.AcceptanceTests/DelayedDelivery/TimeoutManager/When_timeout_dispatch_fails_on_timeout_data_removal.cs

```diff
@@ -123,11 +122,11 @@ public Task RemoveTimeoutBy(Guid sagaId, ContextBag context)
                 public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice)
                 {
                     var timeouts = timeoutData != null
-                        ? new List<TimeoutsChunk.Timeout>
+                        ? new TimeoutsChunk.Timeout[1]
```

Comment:
I don't think you need to specify the array size when using the collection initializer with arrays (other than lists, where it may make sense if the list is a "read only" list)?